extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -624,19 +624,24 @@ export class AmpA4A extends AMP.BaseElement {
           }
           if (fetchResponse.headers && fetchResponse.headers.has(
               EXPERIMENT_FEATURE_HEADER_NAME)) {
-            fetchResponse.headers.get(EXPERIMENT_FEATURE_HEADER_NAME).split(',')
-              .forEach(pair => {
-                const parts = pair.split('=');
-                if (parts.length != 2 || !parts[0]) {
-                  dev().warn(TAG, `invalid experiment feature ${pair}`);
-                  return;
-                }
-                this.postAdResponseExperimentFeatures[parts[0]] = parts[1];
-              });
-            this.cspEnabled_ =
-              this.postAdResponseExperimentFeatures[CSP_ENABLED_EXP_NAME] ==
-                'true';
+            this.populatePostAdResponseExperimentFeatures_(
+                fetchResponse.headers.get(EXPERIMENT_FEATURE_HEADER_NAME));
+          }
+          if (getMode().localDev && this.win.location &&
```

Comment:
Can this.win.location ever be falsey?

Reply:
Its really a guard for test scenarios that don't properly build a full window object.  Given this is behind localDev check and will be compiled out of prod, figured its best to handle edge cases.

Reply:
OK SG